<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control Autorig</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{background-color: rgb(42,72,88);}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">#tabGeneral{
background-color: rgb(42,72,88);
}</string>
    </property>
    <attribute name="title">
     <string>Autorig</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>471</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnCreateControls">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnCreateControls{background-color:rgb(242, 100, 25);}</string>
        </property>
        <property name="text">
         <string>Create Controls</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbFKControls">
        <property name="styleSheet">
         <string notr="true">#rbFKControls{
color:rgb(255,255,255);
}</string>
        </property>
        <property name="text">
         <string>FK Controls Only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFKIKControls">
        <property name="styleSheet">
         <string notr="true">#rbFKIKControls{
color:rgb(255,255,255);
}</string>
        </property>
        <property name="text">
         <string>FK-IK Controls</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">#label_2{color:rgb(255,255,255);}</string>
        </property>
        <property name="text">
         <string>Root joint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnShoulderJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnShoulderJoint{background-color:rgb(134, 187, 216);}</string>
        </property>
        <property name="text">
         <string>&lt;---</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">#label_3{color:rgb(255,255,255);}</string>
        </property>
        <property name="text">
         <string>Shoulder joint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditShoulderJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEditShoulderJoint[text=&quot;&quot;]{
             color: gray;}</string>
        </property>
        <property name="placeholderText">
         <string>Select shoulder joint and press the button</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditRootJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEditRootJoint[text=&quot;&quot;]{
             color: gray;}</string>
        </property>
        <property name="placeholderText">
         <string>Select root joint and press the button</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnRootJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnRootJoint{background-color:rgb(134, 187, 216);}</string>
        </property>
        <property name="text">
         <string>&lt;---</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">#label_4{color:rgb(255,255,255);}</string>
        </property>
        <property name="text">
         <string>Upper Leg joint</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditUpperLegJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEditUpperLegJoint[text=&quot;&quot;]{
             color: gray;}</string>
        </property>
        <property name="placeholderText">
         <string>Select upper leg joint and press the button</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnUpperLegJoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnUpperLegJoint{background-color:rgb(134, 187, 216);}</string>
        </property>
        <property name="text">
         <string>&lt;---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabTransferAnimation">
    <property name="styleSheet">
     <string notr="true">#tabTransferAnimation{
	background-color:rgb(47, 72, 88);
}</string>
    </property>
    <attribute name="title">
     <string>Transfer Animation</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#label{
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>The transfer animation works only for Mixamo animations. This tool will create a reference to the mixamo animation and copy the referenced joint animation to the FK controls of the rig.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectReferenceAnimation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnSelectReferenceAnimation {
   background-color: rgb(134, 187, 216);
}</string>
        </property>
        <property name="text">
         <string>Select Reference Animation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDeleteReference">
        <property name="styleSheet">
         <string notr="true">#chkDeleteReference{
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Delete Referenced Mixamo Animation After Successfully Transferring the Animation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTransferAnimation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnTransferAnimation {
    background-color: rgb(242,100,25);
}</string>
        </property>
        <property name="text">
         <string>Transfer Animation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
